libclamav_rust/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# libclamav features written in Rust
+#
+
+# libclamav rust static library
+add_rust_library(TARGET clamav_rust WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}")
+if (WIN32)
+    target_link_libraries(clamav_rust PUBLIC Userenv)
```

Comment:
I apparently screwed up my recommendation and this actually needs to be:
```cmake
    target_link_libraries(clamav_rust PUBLIC INTERFACE Userenv)
```
With adding "INTERFACE", it seems to work on my windows host.